lib/gems/pending/appliance_console/logfile_configuration.rb

```diff
@@ -102,12 +103,14 @@ def initialize_logfile_disk
     end
 
     def start_evm
+      return true unless evm_was_running
```

Comment:
I think this would read better if we moved this check to the caller.
So in `#activate_new_disk` you would get:
```ruby
start_evm if evm_was_running
```